client/state/sites/selectors/get-jetpack-admin-url.js

```diff
@@ -0,0 +1,15 @@
+import { format as formatUrl, getUrlParts, getUrlFromParts } from '@automattic/calypso-url';
+import getSiteAdminPage from './get-site-admin-page';
+import getSiteAdminUrl from './get-site-admin-url';
+
+export default function getJetpackAdminUrl( state, siteId ) {
```

Comment:
Note for a separate PR, or an extra-small maintenance task: return to `components/jetpack/sidebar` to remove its implementation and replace it with this shared one.

Reply:
I added a task to our board